src/Uno.UI/UI/Xaml/UIElement.Android.cs

```diff
@@ -27,11 +30,33 @@ public UIElement()
 				return;
 			}
 
-			ViewCompat.SetClipBounds(this, rect.LogicalToPhysicalPixels());
+			if (NeedsClipToSlot)
```

Comment:
@carldebilly this viariable is shared crossplatform ? I don't have uno.ui opened right now but it doesn't seem defined in this file.